atlas_direction_vectors/cerebellum.py

```diff
@@ -117,5 +122,5 @@ def cereb_subregion_direction_vectors(
         region_to_weight=region_to_weight,
         shading_width=4,
         expansion_width=8,
-        has_hemispheres=False,
+        has_hemispheres=True,
```

Comment:
yes, this is tricky, I have to figure out how to do, as here we do per lobule, and for example Simple Lobule is not near the center, so it has two parts, one in each hemisphere

Reply:
probably this should be done on a subregion basis, I'll play around with it

Reply:
@drodarie , would this this correct? 

```
INFO:atlas_direction_vectors.app.direction_vectors:Subregion Crus 1 has hemispheres: True
INFO:atlas_direction_vectors.app.direction_vectors:Subregion Crus 2 has hemispheres: True
INFO:atlas_direction_vectors.app.direction_vectors:Subregion Paramedian lobule has hemispheres: True
INFO:atlas_direction_vectors.app.direction_vectors:Subregion Copula pyramidis has hemispheres: True
INFO:atlas_direction_vectors.app.direction_vectors:Subregion Paraflocculus has hemispheres: True
INFO:atlas_direction_vectors.app.direction_vectors:Subregion Lobule II has hemispheres: False
INFO:atlas_direction_vectors.app.direction_vectors:Subregion Lobule III has hemispheres: False
INFO:atlas_direction_vectors.app.direction_vectors:Subregion Nodulus (X) has hemispheres: False
INFO:atlas_direction_vectors.app.direction_vectors:Subregion Folium-tuber vermis (VII) has hemispheres: False
INFO:atlas_direction_vectors.app.direction_vectors:Subregion Uvula (IX) has hemispheres: False
INFO:atlas_direction_vectors.app.direction_vectors:Subregion Declive (VI) has hemispheres: False
INFO:atlas_direction_vectors.app.direction_vectors:Subregion Lobules IV-V has hemispheres: False
INFO:atlas_direction_vectors.app.direction_vectors:Subregion Flocculus has hemispheres: True
INFO:atlas_direction_vectors.app.direction_vectors:Subregion Simple lobule has hemispheres: True
INFO:atlas_direction_vectors.app.direction_vectors:Subregion Lingula (I) has hemispheres: False
INFO:atlas_direction_vectors.app.direction_vectors:Subregion Pyramus (VIII) has hemispheres: False
```
I detect hemispheres if they have voxel at the halfplane +- 1 voxel.

Reply:
The has_hemisphere feature was created to prevent an orientation field that is computed on one hemisphere to affect the other one. But this has an effect only when they directly face each other and are very close to each other. This is the case for isocortex but not for the cerebellar cortex because we deal with each subregion individually.
I believe your change would have close to no effect here. More, this means that you need to compute the orientation field twice for each hemispherical subregion. 
IMO, there is a lack of a grounded explanation for this change.